src/main/java/net/fabricmc/fabric/impl/launch/APIWarning.java

```diff
@@ -16,11 +16,15 @@
 
 package net.fabricmc.fabric.impl.launch;
 
+import java.util.Locale;
+import java.util.ResourceBundle;
+
 import javax.swing.JOptionPane;
 
 public final class APIWarning {
 	public static void main(String[] args) {
-		String st = "Fabric API is not meant to be run.\nPlease place this file in your mods folder.";
-		JOptionPane.showMessageDialog(null, st);
+		Locale defaultLocale = Locale.getDefault();
+		ResourceBundle WarningAPI = ResourceBundle.getBundle("lang/WarningAPI", defaultLocale);
```

Comment:
I hope we can have a bundle setup like https://github.com/FabricMC/fabric-installer/blob/e76d587e22ae9767c0f97fc782c991e461881b1c/src/main/java/net/fabricmc/installer/util/Utils.java#L36-L50 so that we don't need to use iso encoding and ugly unicode escapes for the translation properties